lib/ingres_sa_dialect/ingresdbi.py

```diff
@@ -17,9 +18,14 @@ class Ingres_ingresdbi(IngresDialect):
     def __init__(self, **kwargs):
         IngresDialect.__init__(self, **kwargs)
 
-    @classmethod
-    def dbapi(cls):
-        return __import__('ingresdbi')
+    if (sqlalchemy_version_tuple >= (2,0)):
+        @classmethod
+        def import_dbapi(cls):
```

Comment:
as per last comment, lets get a ticket opened for potential refactor here and push logic into base (if possible)